Assets/Scripts/Game/PlayerMouseLook.cs

```diff
@@ -4,9 +4,9 @@
 // License:         MIT License (http://www.opensource.org/licenses/mit-license.php)
 // Source Code:     https://github.com/Interkarma/daggerfall-unity
 // Original Author: Gavin Clayton (interkarma@dfworkshop.net)
-// Contributors:    
+// Contributors:    Avernite (avernite@gmail.com)
 // 
-// Notes:
+// Notes:           Look-smoothing code changed in 0.14.5 (Avernite)
```

Comment:
No need to add this note here (that's what git is for). Also, this change won't be in 0.14.5. :)

Reply:
Note removed.